cli/common/lua.go

```diff
@@ -67,19 +68,16 @@ func LuaGetRocksVersions(appDirPath string) (map[string]string, error) {
 				depInfoLTable.ForEach(func(depVersionL lua.LValue, _ lua.LValue) {
 					depVersion := depVersionL.String()
 					if _, found := rocksVersionsMap[depName]; found {
-						log.Warnf(
-							"Found multiple versions for %s dependency in rocks manifest",
-							depName,
-						)
+						rocksDuplicates = append(rocksDuplicates, depName)
 					}
 					rocksVersionsMap[depName] = depVersion
 				})
 			}
 		})
 
 	} else if !os.IsNotExist(err) {
-		return nil, fmt.Errorf("Failed to read manifest file %s: %s", manifestFilePath, err)
+		return nil, nil, fmt.Errorf("Failed to read manifest file %s: %s", manifestFilePath, err)
 	}
 
-	return rocksVersionsMap, nil
+	return rocksVersionsMap, rocksDuplicates, nil
```

Comment:
Good. Let's make it better - return one value, a structure `RocksInfo` that contains `Versions` map and `Duplicates` strings array. The function become `getRocksInfo`.